src/examples/button/SplitButtonDanger.tsx

```diff
@@ -5,33 +5,34 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import React, { useState } from 'react';
+import React, { useCallback, useState } from 'react';
 import { Row, Col } from '@zendeskgarden/react-grid';
 import { SplitButton, Button, ChevronButton } from '@zendeskgarden/react-buttons';
-import { Dropdown, Trigger, Menu, Item } from '@zendeskgarden/react-dropdowns';
+import { Menu, Item } from '@zendeskgarden/react-dropdowns.next';
 
 const Example = () => {
   const [rotated, setRotated] = useState<boolean>();
 
+  const handleChange = useCallback(changes => {
+    changes.isExpanded !== undefined && setRotated(changes.isExpanded);
+  }, []);
+
   return (
     <Row>
       <Col textAlign="center">
         <SplitButton>
           <Button isDanger>Harvest</Button>
-          <Dropdown
-            onStateChange={options =>
-              Object.hasOwn(options, 'isOpen') && setRotated(options.isOpen)
-            }
+          <Menu
+            button={props => (
+              <ChevronButton {...props} aria-label="other actions" isRotated={rotated} isDanger />
+            )}
```

Comment:
We'll be able to eradicate these with the upcoming `Menu` minor release, right?

Reply:
Looks like we have two options here as far as recommended API usage.

Option 1:

The change currently in this PR.

Option 2:

```
const Example = () => (
  <Row>
    <Col textAlign="center">
      <SplitButton>
        <Button isDanger>Harvest</Button>
        <Menu button="" buttonProps={{ isDanger: true }} placement="bottom-end">
          <Item value="prune">Prune</Item>
          <Item value="water">Water</Item>
          <Item value="fertilize">Fertilize</Item>
        </Menu>
      </SplitButton>
    </Col>
  </Row>
);
```

---

The reason for the difference is that `button` is a required prop - but can be left as an empty string to fulfill its required typing, while minimizing the additional state logic. Option 1 feels weird but is possible & has some minimalist benefits.

Reply:
Another fair point might be to remove required condition on `button` for this exact use-case.

Reply:
Ok, I'll look to you to go with whatever approach is cleanest.

Reply:
I ended up leaving this code as-is. Menu is using `Button.IconEnd` which adds space when `button` is an empty string. It will be the same implementation as before, at least. :)